index.html

```diff
@@ -138,16 +138,16 @@ <h3>Profesional Art Printing Data</h3>
                 </div>
             </div>
         </section>
-        <section class="about">
+        <section id="about" class="about">
             <div class="gen-desc">
                 <h2 class="heading-text">About me</h2>
-                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <p class="paragraph-text">Hello I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 <button class="regular-btn">Get My Resume</button>
```

Comment:
- [ ] Your button has not had enough contrast of 4.5:1. Please add enough contrast for this button :+1: 

![Screenshot from 2023-01-25 00-12-18](https://user-images.githubusercontent.com/80458199/214433342-db05a5d2-8232-4812-9326-7fd6c9a9cbc1.png)

![Screenshot from 2023-01-25 00-13-08](https://user-images.githubusercontent.com/80458199/214433357-1b6fa601-52a5-432a-872c-35841658ceea.png)